src/applications/caregivers/components/ApplicationDownloadLink.jsx

```diff
@@ -67,7 +67,7 @@ const ApplicationDownloadLink = ({ formConfig }) => {
         handlePdfDownload(blob);
         recordEvent({ event: 'caregivers-10-10cg-pdf-download--success' });
       } catch (error) {
-        setErrors(error.errors || []);
+        setErrors(error.errors);
```

Comment:
We don't want this set to an empty array. That would indicate from a UI perspective there are no errors